addons/l10n_it_edi/__init__.py

```diff
@@ -3,3 +3,23 @@
 
 from . import models
 from . import tools
+
+from odoo import api, SUPERUSER_ID
+
+
+def _l10n_it_edi_update_export_tax(env):
```

Comment:
Is this also handled when you install a new CoA after installing this module?

Reply:
No, because these fields are not on tax.templates. We'll solve this problem later. People can still configure this tax properly manually.